### ISSUE TITLE
Dart Client

### ISSUE THREAD

**Issue by ghost** [celrenheit/sandglass-client#4]
Dart grpc works with golang grpc, so we should be easily able to gen a Dart client.

I had it working for a test but the Dart grpc team recently broke things. 
Please leave open for now ??

**Comment by celrenheit** [comment]
Yes I'll leave it open. However I'm gonna split this into a separate package, probably named sandglass-dart. Please check issue #5 to track the progress.
